postpic/particles/particles.py

```diff
@@ -1299,18 +1299,41 @@ def exporttoh5(self, filename, *scalarfs):
         - particles/Id/track: track data (id, weight, *scalarfs)
         - particles/Id/weight: weights
         '''
-        scalarfs = ("id", "weight") + scalarfs
+        scalarfs = ("weight", ) + scalarfs
         part_data = self.collect(*scalarfs)
 
         import h5py
 
         with h5py.File(filename, "w") as f:
             dset_ids = f.create_dataset("Ids", data=self.ids, dtype=np.int64)
-            f.attrs['scalars'] = scalarfs
+            f.attrs['PostPic_scalars'] = scalarfs[1:]
 
             for i in range(0, len(self.ids)):
-                f.create_dataset("particles/{}/track".format(int(self.ids[i])), data=part_data[i])
-                weight = part_data[i][1][0]
-                f.create_dataset("particles/{}/weight".format(int(self.ids[i])), data=weight)
+                f.create_dataset("particles/{}/track".format(int(self.ids[i])),
+                                 data=part_data[i][1:][:])
+                weight = part_data[i][0][0]
+                f.create_dataset("particles/{}/weight".format(int(self.ids[i])),
+                                 data=weight)
 
         return part_data
+
+    def exporttoh5_hyena(self, filename):
+        '''
+        Exports the collected data to an hdf5-file for the hyena-format.
+
+        Parameters:
+        -----------
+        filename:   whole name or path of the file which the data will be exported to.
+
+        Returns:
+        -----------
+        The same as collect(): The list of particles with the defined properties.
+        The hdf5 file will have the structure:
+        - dset: Ids
+        - particles/Id/track: track data ("time*c*1e6", "x_um", "y_um", "z_um",
+            "px/m_e/c", "py/m_e/c", "pz/m_e/c")
+        - particles/Id/weight: weights
+        '''
+
+        return self.exporttoh5(filename, "time*c*1e6", "x_um", "y_um",
+                               "z_um", "px/m_e/c", "py/m_e/c", "pz/m_e/c")
```

Comment:
warum wird px auf m_e normiert? Ist das der Speziallfall dass es nur fuer Elektronen funktioniert? Kannst du nicht vielleicht auf `mass` normieren, also auf die tatsaechliche masse des teilchens und dann klappt es auch fuer ionen? Alternativ in den Hilfetext eine Warnung in Grossbuchstaben dass hyena nur elektronen versteht -- falls das so ist. das weiss ich nicht.